Description: create namespace package ‘sphinxcontrib’
 Create namespace package ‘sphinxcontrib’. This allows python-sphinxcontrib.*
 packages, both those using dh_python2 and those using python-support, to be
 co-importable.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-11-20

--- a/sphinx/__init__.py
+++ b/sphinx/__init__.py
@@ -35,6 +35,19 @@
     except Exception:
         pass
 
+def _create_sphinext_namespace():
+    # Create namespace package "sphinxcontrib".
+    import pkgutil
+    module_type = type(sys)
+    try:
+        sphinxcontrib = sys.modules['sphinxcontrib']
+    except LookupError:
+        sphinxcontrib = module_type('sphinxcontrib')
+        sphinxcontrib.__path__ = []
+        sys.modules[sphinxcontrib.__name__] = sphinxcontrib
+    sphinxcontrib.__path__ = pkgutil.extend_path(sphinxcontrib.__path__, sphinxcontrib.__name__)
+
+_create_sphinext_namespace()
 
 def main(argv=sys.argv):
     """Sphinx build "main" command-line entry."""
